scripts/orchestrators/events.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/env bash
+
+# Includes
+source ./_helpers.sh
+
+azlogin "${ARM_SUBSCRIPTION_ID}" "${ARM_TENANT_ID}" "${ARM_CLIENT_ID}" "${ARM_CLIENT_SECRET}" 'AzureCloud'
+
+# expects EVENTS_STORAGE_ACCOUNT, EVENTS_TABLE_NAME to be set
+
+usage() {
+    _information "Usage: helpers to store and query events"
+    exit 1
+}
+
+query_events() {
+    local pipeline_name=$1
+    local event_name=$2
+    local event_group_id=$3
+
+    filter_expression="${odata_filter} PartitionKey eq '${pipeline_name}'"
+
+    cmd="az storage entity query \
```

Comment:
this function would need  to check the exit code after the az cmd exeution to avoid the pipeline silently failing with false green state [see here](https://github.com/hadwagaber/symphony_test/actions/runs/6435926443/job/17478231608?pr=3)

Reply:
Yes! It was there and I removed it for testing purposes. Forgot to add it back.

Reply:
Done